modules/manage/pages/cluster-maintenance/manage-throughput.adoc

```diff
@@ -25,29 +23,20 @@ The properties for broker-wide throughput quota balancing are configured at the
 | A broker's total throughput limit for egress Kafka traffic.
 
-| xref:reference:cluster-properties.adoc#kafka_quota_balancer_node_period_ms[kafka_quota_balancer_node_period_ms]
-| The period at which the quota balancer runs to balance throughput quota between a broker's shards.
-
-| xref:reference:cluster-properties.adoc#kafka_quota_balancer_min_shard_throughput_ratio[kafka_quota_balancer_min_shard_throughput_ratio]
-| The lowest value of the throughput quota a shard can get in the process of quota balancing, expressed as a ratio of the default shard quota. If set as `0`, there is no minimum, and if set as `1`, no quota can be taken away by the balancer.
-
-| xref:reference:cluster-properties.adoc#kafka_quota_balancer_min_shard_throughput_bps[kafka_quota_balancer_min_shard_throughput_bps]
-| The lowest value of the throughput quota a shard can get in the process of quota balancing, in bytes per second. If set as `0`, there is no minimum.
-
 | xref:reference:tunable-properties.adoc#max_kafka_throttle_delay_ms[max_kafka_throttle_delay_ms]
 | The maximum delay inserted in the data path of Kafka API requests to throttle them down. Configuring this to be less than the Kafka client timeout can ensure that the delay that's inserted won't be long enough to cause a client timeout by itself.
 
-| xref:reference:cluster-properties.adoc#kafka_quota_balancer_window_ms[kafka_quota_balancer_window_ms]
-| The time window the balancer uses to average the current throughput measurement.
 |===
 
 [NOTE]
 ====
 * By default, both `kafka_throughput_limit_node_in_bps` and `kafka_throughput_limit_node_out_bps` are disabled, no throughput limits are applied. You must manually set them to enable quota balancing with throughput limits.
-* `kafka_quota_balancer_min_shard_throughput_bps` doesn't override the `kafka_throughput_limit_node_in_bps` and `kafka_throughput_limit_node_out_bps` limit settings. Consequently, the value of
-`kafka_throughput_limit_node_in_bps` or `kafka_throughput_limit_node_out_bps` can result in lesser throughput than `kafka_quota_balancer_min_shard_throughput_bps`.
 ====
 
+=== Advanced broker-wide throughput limit tuning
+
+The throttling mechanism separates ingress and egress quotas and applies them across all CPU cores. When one of the throttling quotas is reached, Redpanda adds a throttling delay, `throttle_time_ms`, to subsequent responses. If the client still fails to back off, then the next read operation is delayed. Redpanda attempts to set the throttling delay to a number that will put the cluster back below the throttling threshold. This delay will not exceed the value set in `max_kafka_throttle_delay_ms`. Administrators may also use the tunable property  xref:reference:tunable-properties.adoc#kafka_throughput_replenish_threshold[`kafka_throughput_replenish_threshold`] to adjust the rate that Redpanda eases throttling once it is engaged.
```

Comment:
What are those values? cluster properties? if yes, please link to them

Reply:
There's not one for `throttle_time_ms` since it's a property in the response object. I did add a link for `max_kafka_throttle_delay_ms`.